package-lock.json

```diff
@@ -1,12460 +1,8 @@
 {
   "name": "h",
   "version": "0.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```

Comment:
Not sure what'ds going on here, you might have taken a shortcut to remove lots of lines?

Reply:
Eugh. This is a really annoying file that gets regenerated all the time and makes a mess of the checkout status. So I've committed it by mistake. I've fixed this now.